makefile

```diff
@@ -1,8 +1,25 @@
-clean:
+# Copyright 2019 AccelByte Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ clean:
 	rm coverage.out
 
 test:
-	go test -cover ./...
+	docker-compose -f docker-compose-test.yml up -d
+	sleep 30
```

Comment:
What's the reason we have this?

Reply:
I think it is waiting for the docker-compose service available

Reply:
Yeah, that's correct. It's for witing the docker-compose to be ready.